packages/prci/src/mixins/with-computation-api.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable max-classes-per-file */
+import { LitElement } from 'lit';
+import { property } from 'lit/decorators.js';
+import { ComputationAPI } from '../utils/computation-api.js';
+
+type Constructor<T> = new (...args: any[]) => T;
+
+export declare class WithComputationApiInterface {
+  computationApi: ComputationAPI;
+
+  computationBaseURL: string;
+}
+
+export const WithComputationApi = <T extends Constructor<LitElement>>(
```

Comment:
Good call :+1: For now, I'm only focusing on the PRCI to avoid a too big PR.
This is indeed something we should address after merging it (as we need to reproduce the same approach in the DCI).

Reply:
This PR doesn't include a mixin anymore. Moving the ComputationAPI class to another library shared with the DCI package is the next step (see https://github.com/blindnet-io/devrel-management/issues/81).